client/src/components/SubmittedDesigns/useStyles.tsx

```diff
@@ -0,0 +1,7 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
```

Comment:
Are we going to be using this in the future?

Reply:
i deleted this file for now; can create in the future if needed